rules/review-videos-collaboratively/rule.md

```diff
@@ -0,0 +1,110 @@
+---
+type: rule
+title: Do you use dedicated collaborative software to review videos?
+uri: review-videos-collaboratively
+authors:
+  - title: William Greentree
+    url: https://ssw.com.au/people/alumni/will
+    img: https://avatars.githubusercontent.com/u/109055250?v=4
+created: 2023-03-06T05:51:57.440Z
+guid: 5d66af76-53e0-43da-9bd5-23dcab2c3892
+---
+So you have just finished that sweet video edit and you are now ready & hyped to receive feedback from your team and or client. You just sent the link to the video to get "Test Pleased" but the feedback is hard to read and or follow. This is due to the following:
+
+❌ Feedback is disconnected from the video player\
```

Comment:
Hey @williamgreentreessw 

1. Can you re-word these 2 lists to follow: https://ssw.com.au/rules/avoid-repetition/

Reply:
Done